src/v3/src/util/passwordUtils.ts

```diff
@@ -136,6 +137,24 @@ const excludeLastNameValidator = (
     : excludeAttributeValidator(password, ruleVal as boolean, userInfo.profile.lastName)
 );
 
+const useADComplexityRequirementsValidator = (
+  password: string,
+): boolean => {
+  const requiredValidators: ((password: string, limit: unknown) => boolean)[] = [
+    minLowerCaseValidator,
+    minUpperCaseValidator,
+    minNumberValidator,
+    minSymbolValidator,
+  ];
+
+  // Runs the 4 validation functions and returns the number that passed
+  const numRequirementsMet = requiredValidators.reduce((num, validator) => (
+    validator(password, 1) ? num + 1 : num), 0);
```

Comment:
Calls each of the 4 possible validators that can satisfy AD requirements with a hard-coded `limit=1`